hcx/README.md

```diff
@@ -0,0 +1,32 @@
+# HCX-ESLZ
+
+[What is HCX](whatishcx.md)
```

Comment:
This should probably not be a link and be part of the readme

Reply:
Updated